backend/pipelines/bmd_scraper/.env.example

```diff
@@ -0,0 +1,18 @@
+# BMD Portal Configuration
+BMD_BASE_URL=https://bmd.mst.dk
+
+# Directory Settings
+BRONZE_OUTPUT_DIR=/data/bronze/bmd
```

Comment:
this saved it in my root folder instead of in /backend as is the case with the other two pipelines.